### ISSUE TITLE
OpenGL ES 3 via Angle

### ISSUE THREAD

**Issue by l3m** [opentk/opentk#208]
There are dlls and demos for using OpenGL ES 2 via [Angle](https://code.google.com/p/angleproject/), but none for OpenGL ES 3. 

Is this coming, or what is the plan here?

**Comment by thefiddler** [comment]
Yes, they will be coming on the next update.

ANGLE is actually pretty easy to compile using Visual Studio (see [Building Angle](https://code.google.com/p/angleproject/wiki/DevSetup).)

**Comment by l3m** [comment]
I built a custom angle from source, and tried to setup a demo, using [this](https://github.com/thefiddler/opentk-wpf) project. 

- Copied over the self-compiled binaries to opentk dependencies
- Recompiled the opentk develop branch
- Copied over both the new OpenTK dlls as well as the Angle dlls to the opentk-wpf project



Then I compiled opentk-wpf. When I try to run it, I get an exception: 

    'OpenTK.Graphics.GraphicsContextException' occurred in OpenTK.dll
    Additional information: Failed to create EGL context, error: BAD_CONFIG.



I also noticed there's an (unmerged) ES31 branch in the opentk repo. 

So my questions are: 

- Did I forget anything important? 
- Is current Angle supposed to work with OpenTk develop (or latest release), or is some ES31 merging necessary? 

My attempts can be seen here [here](https://github.com/l3m/opentk-wpf/commits/angle-from-source)
